test/conftest.py

```diff
@@ -81,7 +81,8 @@ def insert_data_from_csv(in_csv_file, rec_count=-1):
                       stats=True,
                       )
         assert loader.error_counter == 0
-        return data
+        # Preserve loader so that dask manager is not auto deleted.
+        return data, loader
```

Comment:
I guess this is the reason for the change above? Why do we need to preserve the dask manager?

Reply:
It encapsulates the Localcluster, and client.

Reply:
Ok, sounds good, can you just add something that explains below so someone can know looking at the code later?

great detective work